Performance test ScaleFrom050 is failing

/area test-and-release
/kind bug

Error:
I0129 00:58:04.709] --- FAIL: TestScaleFromZero50 (222.65s)
I0129 00:58:04.709]     scale_from_zero_test.go:156: Failed waiting for deployment to scale to zero: timed out waiting for the condition

Build log:
https://storage.googleapis.com/knative-prow/pr-logs/pull/knative_serving/3017/pull-knative-serving-perf-tests/1090046025596407808/build-log.txt

--- a/test/performance/scale_from_zero_test.go
+++ b/test/performance/scale_from_zero_test.go
@@ -237,7 +237,5 @@ func TestScaleFromZero5(t *testing.T) {
 }
 
 func TestScaleFromZero50(t *testing.T) {
-	// See: #3021
-	t.Skip()
 	testScaleFromZero(t, 50 /* parallelism */, 5 /* runs */)
 }

